nuxeo-features/rest-api/nuxeo-rest-api-test/src/test/java/org/nuxeo/ecm/restapi/test/PreviewAdapterTest.java

```diff
@@ -80,7 +80,7 @@ public void testFileAttachmentPreview() {
         TransactionHelper.commitOrRollbackTransaction();
         TransactionHelper.startTransaction();
 
-        try (CloseableClientResponse response = getPreview(doc, "files/0/file")) {
+        try (CloseableClientResponse response = getPreview(doc, "files:files/0/file")) {
             assertEquals(200, response.getStatus());
```

Comment:
The REST BlobObject now requires canonical xpaths with prefixes, so that the security filters that want to check xpaths only have the canonical xpaths to check, and not several variations. But I realize that the changes to enforce canonical xpaths (and normalize blobholder:0 to the correct file:content or whatever) are incomplete. I'll push more changes.